### DISCUSSION TITLE
How can I configure @wagmi/core provider?

### DISCUSSION THREAD

**Discussion by akinncar** [wevm/wagmi#1673] [Q&A]
Look at the docs about @wagmi/core config
https://wagmi.sh/core/getting-started
```js
import {
  WagmiConfig,
  createClient,
  configureChains,
  mainnet,
} from '@wagmi/core'
```
it recommends to import `WagmiConfig` from `@wagmi/core`
but there's no `WagmiConfig` in `@wagmi/core`
and also it's not used on the docs
I was expecting something like `wagmi` usage
```js
import { WagmiConfig } from 'wagmi'
 
function App() {
  return (
    <WagmiConfig>
      <YourRoutes />
    </WagmiConfig>
  )
}
```

I tried to import from `wagmi` directly but mix `@wagmi/core` and `wagmi` doesn't work

you can recommend me to use `wagmi` directly since I'm using react, but there's anyway to use [Actions](https://wagmi.sh/core/actions/watchAccount) for example, without `@wagmi/core`?

**Comment by tmm** [comment]
`WagmiConfig` should not be there (will update the docs). For `@wagmi/core`, all you need to do is set up the Client in an entry file. If you are using React, I would recommend using `wagmi` (you can import Vanilla JS actions from the `wagmi/actions` entrypoint).
